### PR TITLE
[new release] merlin-extend (0.6.1)

### PR DESCRIPTION
**Pull request by let-def**
A protocol to provide custom frontend to Merlin

- Project page: <a href="https://github.com/let-def/merlin-extend">https://github.com/let-def/merlin-extend</a>
- Documentation: <a href="https://let-def.github.io/merlin-extend">https://let-def.github.io/merlin-extend</a>

##### CHANGES:

Add missing dependency on unix (fix by @dra27)

### CODE REVIEW

File: packages/merlin-extend/merlin-extend.0.6.1/opam [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+bug-reports: "https://github.com/let-def/merlin-extend"
+license: "MIT"
+dev-repo: "git+https://github.com/let-def/merlin-extend.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "dune" {>= "1.0"}
+  "cppo" {build}
```
Comment (kit-ty-kate) [inline packages/merlin-extend/merlin-extend.0.6.1/opam:14]: Uses `cppo -V`
```suggestion
  "cppo" {build & >= "1.1.0"}
```

File: packages/merlin-extend/merlin-extend.0.6.1/opam [hunk 2]
```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+maintainer: "Frederic Bour <frederic.bour@lakaban.net>"
+authors: "Frederic Bour <frederic.bour@lakaban.net>"
+homepage: "https://github.com/let-def/merlin-extend"
+bug-reports: "https://github.com/let-def/merlin-extend"
+license: "MIT"
+dev-repo: "git+https://github.com/let-def/merlin-extend.git"
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (kit-ty-kate) [inline packages/merlin-extend/merlin-extend.0.6.1/opam:9]:
```suggestion
  ["dune" "subst"] {dev}
```

**Comment by kit-ty-kate** [comment]
Thanks. Could you return the fixes to the opam file upstream?

**Comment by let-def** [comment]
Done thank you!

**Pull request merged**